stock_available_product_expiry/models/stock_quant.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class StockQuant(models.Model):
+
+    _inherit = 'stock.quant'
+
+    lot_id = fields.Many2one(
+        'stock.production.lot',
+        auto_join=True
```

Comment:
@rousseldenis IMO the auto_join works only with required fields. Can you check that quants without lot and reservation_id are still visible with `auto_join=True`?

Reply:
Ok, I'll check

Reply:
Indeed, not visible... :-(

I try to find another way to compute